app/models/chef.rb

```diff
@@ -1,4 +1,8 @@
 class Chef < ApplicationRecord
-   validates_presence_of :name
-   has_many :dishes
+  validates_presence_of :name
+  has_many :dishes
+
+  def uniq_ingredients
+    Ingredient.distinct.pluck(:name)
```

Comment:
For this to work correctly, you'll need to set up an additional association between chefs and ingredients here: `has_many :ingredients, through: :dishes` --- leveraging the relationships that dish has already set up. Otherwise, this query won't appropriately pull the ingredients just for this chef.